src/test/java/DecodeTest.java

```diff
@@ -0,0 +1,73 @@
+import decoder.Decoder;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Validate that decoding the protobuf file to json works properly. */
```

Comment:
Use //.
Or start a new line.

Reply:
The autoformatter makes this the style. I agree that it doesn't look nice. I originally had it the way you suggested here.